src/adapters/mc/OsConfigResource.c

```diff
@@ -168,6 +168,8 @@ void MI_CALL OsConfigResource_Load(
         CallMpiClose(g_mpiHandle, GetLog());
         g_mpiHandle = NULL;
     }
+
+    SetConsoleLoggingEnabled(false);
```

Comment:
>SetConsoleLoggingEnabled(false); [](http://example.com/codeflow?start=4&length=32)

This line is already in dev branch. Why does appear as newly added here? Did you merge dev?